ui/src/components/console/worker.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-ignore
+self.MonacoEnvironment = {
```

Comment:
these workers still dont actually work.. im going to create an issue once this is merged to see if anyone with more knowledge of react/vite/monaco/web workers can help

Reply:
yah i did, i think those docs are from an older version of vite, like 2.0 and they don't seem to work for the later version we are using , 4.0